src/main/java/uk/gov/migration/R__11_Migrate_current_keys_to_index.java

```diff
@@ -0,0 +1,220 @@
+package uk.gov.migration;
+
+import io.dropwizard.jdbi.OptionalContainerFactory;
+import org.flywaydb.core.api.migration.MigrationChecksumProvider;
+import org.flywaydb.core.api.migration.jdbc.BaseJdbcMigration;
+import org.skife.jdbi.v2.DBI;
+import org.skife.jdbi.v2.Handle;
+import uk.gov.register.configuration.IndexFunctionConfiguration;
+import uk.gov.register.core.*;
+import uk.gov.register.db.DerivationRecordIndex;
+import uk.gov.register.db.EntryQueryDAO;
+import uk.gov.register.db.IndexDAO;
+import uk.gov.register.db.IndexQueryDAO;
+import uk.gov.register.indexer.IndexDriver;
+import uk.gov.register.indexer.function.RecordIndexFunction;
+import uk.gov.register.store.DataAccessLayer;
+import uk.gov.register.store.postgres.PostgresDataAccessLayer;
+import uk.gov.register.util.HashValue;
+import uk.gov.register.views.ConsistencyProof;
+import uk.gov.register.views.EntryProof;
+import uk.gov.register.views.RegisterProof;
+
+import java.sql.Connection;
+import java.time.Instant;
+import java.util.*;
+
+public class R__11_Migrate_current_keys_to_index extends BaseJdbcMigration implements MigrationChecksumProvider {
+	@Override
+	public void migrate(Connection connection) throws Exception {
+		TempDataSource ds = new TempDataSource(connection);
+		DBI dbi = new DBI(ds);
+		dbi.registerContainerFactory(new OptionalContainerFactory());
+		Handle handle = dbi.open();
+		
+		DataAccessLayer dataAccessLayer = new PostgresDataAccessLayer(
+				handle.attach(EntryQueryDAO.class),
+				handle.attach(IndexDAO.class),
+				handle.attach(IndexQueryDAO.class),
+				null,
```

Comment:
Is it possible to not make these `null`? Seems a bit nasty to leave them as `null`.